op-service/txintent/interop_call.go

```diff
@@ -0,0 +1,145 @@
+package txintent
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/constants"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/lmittmann/w3"
+
+	suptypes "github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+)
+
+var _ Call = (*InitTrigger)(nil)
+var _ Call = (*ExecTrigger)(nil)
+
+var _ Call = (*SendTrigger)(nil)
+var _ Call = (*RelayTrigger)(nil)
+
+// Trigger for using the EventLogger to trigger emitLog
+type InitTrigger struct {
+	Emitter    common.Address // address of the EventLogger
+	Topics     [][32]byte
+	OpaqueData []byte
+}
+
+func (v *InitTrigger) To() (*common.Address, error) {
+	return &v.Emitter, nil
+}
+
+func (v *InitTrigger) Data() ([]byte, error) {
+	// TODO: Need to do better construct call input than this
```

Comment:
We have a policy to always add an issue-number to TODO comments, so we know work is tracked. I am not sure why CI didn't pick this up, but let's open a github issue.

Reply:
Added at https://github.com/ethereum-optimism/optimism/pull/14948/commits/02952dddccb2c22d5fa169ffac797e327f1b0243